.env.development

```diff
@@ -5,3 +5,4 @@ REDIS_URL=redis://localhost:6379
 SIDEKIQ_PASSWORD=password
 SIDEKIQ_USERNAME=abtion
 USE_LETTER_OPENER=true
+LOCALE=da
```

Comment:
I'm moving this to `.env` so we don't get surprises when we deploy to the production environment